File: OP2/OP2lab1/C++/Laba1/Laba1/Laba1.cpp
#include "Header.h"
#include <iostream>
using namespace std;
int main()
{
	char Text1[] = "text1.txt";
	char Text2[] = "text2.txt";
	cout << "   Enter - end of line\n   Ctrl + S - end of file (after Ctrl + S enter Enter to finish writing the text)\n";
	write(Text1);
	cout << "   If you want to add to file press Y, else - press any other symbol and press Enter: ";
	char Question;
	cin >> Question;
	add(Text1, Question);
	cout << "   Full text of file 1:\n";
	out(Text1);
	cout << "   Enter ammount of symbols:";
	int amm;
	cin >> amm;
	cout << "   Full text of file 2:\n";
	fill(Text1, Text2, amm);
	out(Text2);
	return 0;
	system("pause");
}

File: OP2/OP2lab1/Python/Laba1/Laba1/Laba1.py
from Header import *

print("   Enter - end of line\n   Ctrl + S - end of file (after Ctrl + S enter Enter to finish writing the text)")
put("file1.txt")
a=input("\n   If you want to add to file press Y, else - press any other symbol and press Enter: ")
add("file1.txt", a)
print("\n   Full text of file 1:")
out("file1.txt")
qua=input("   Enter ammount of symbols:")
fil("file1.txt", "file2.txt", qua)
print("\n   Full text of file 2:")
out("file2.txt")